src/checksum/arm64/crc32-arm64.c

```diff
@@ -59,30 +59,45 @@ static uint32_t
 __wt_checksum_with_seed_hw(uint32_t seed, const void *chunk, size_t len)
 {
     uint32_t crc;
-    size_t nqwords;
-    const uint8_t *p;
     const uint64_t *p64;
+    const uint32_t *p32;
+    const uint16_t *p16;
+    const uint8_t *p8;
 
     crc = ~seed;
 
     /* Checksum one byte at a time to the first 4B boundary. */
-    for (p = chunk; ((uintptr_t)p & (sizeof(uint32_t) - 1)) != 0 && len > 0; ++p, --len) {
-        CRC32CB(crc, *p);
+    for (p8 = chunk; ((uintptr_t)p8 & (sizeof(uint32_t) - 1)) != 0 && len > 0; ++p8, --len)
+        CRC32CB(crc, *p8);
```

Comment:
I think we should remove this block. It is weird anyway that we would try to align to a 4-byte boundary, then proceed to do 8 byte reads.

Reply:
Does the hardware instruction works on unaligned addresses?